graylog2-web-interface/src/views/logic/fieldactions/ChangeFieldType/ChangeFieldTypeModal.tsx

```diff
@@ -132,9 +132,14 @@ const ChangeFieldTypeModal = ({ show, onClose, field }: Props) => {
                         required />
         </Input>
         <Alert bsStyle="info">
-          By default the type will be changed in all index sets of the current message/search. By expanding the next section, you can select for which index sets you would like to make the change.
+          By default the <b>{newFieldType}</b> as a field type for <b>{field}</b> will be changed in all index sets of the current message/search. You can select for which index sets you would like to make the change.
```

Comment:
When there is no field type selected it reads like this "By default the as a field type for **action** will be changed in all index sets"
When a field type got selected, the sentence it still reads strange: "By default the **boolean** as a field type for **action** will be changed in all index sets "